collectors/okta/okta_collector.js

```diff
@@ -31,13 +31,12 @@ class OktaCollector extends PawsCollector {
         const startTs = process.env.paws_collection_start_ts ? 
                 process.env.paws_collection_start_ts :
                     moment().toISOString();
-        const endTs = moment(startTs).add(this.pollInterval, 'seconds').toISOString();
-        const initialState = {
+        const initialState = this._getNextCollectionState({
             since: startTs,
-            until: endTs,
+            until: startTs,
```

Comment:
from startTs to startTs? the initial state will have a 1 second time period?

Reply:
Which of the 3 states does this refer to?

Reply:
This is a way to reuse `this._getNextCollectionState()` in order to get initial state. And `since: startTs` is not actually used by `_getNextCollectionState()` function.
The state 
```
{
            since: startTs,
            until: startTs
}
```
is sort of state 0 which is used as a state preceding the initial one